frontend/src/setupProxy.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-undef */
+const { createProxyMiddleware } = require('http-proxy-middleware')
+
+// Added a proxy middleware as api routes with text/html in its Accept header
+// were not getting redirected to the proxy
```

Comment:
Clarification on the behaviour (+ GammarNazi 😜)

```suggestion
// Added a proxy middleware, as requests to api routes with text/html in their Accept header
// were not getting proxied to the backend
```

Reply:
hahaha thanks tim, [aa46d3c](https://github.com/opengovsg/FormSG/pull/5837/commits/aa46d3c02b7afecbf6c75c33cca9b9bfb545f0b1)